js/popup.js

```diff
@@ -0,0 +1,116 @@
+const store = {
+
+  detail1: {
+    name: 'Tonic',
+    img: 'https://codecaiine.github.io/microverse-javascript/image/portfolio1.png',
+    companyName: 'CANOPY',
+    job: 'Back End Dev',
+    year: 2015,
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    tags: ['html', 'css', 'javaScript'],
```

Comment:
* Kindly add the link to the live version and link to source in each object as it is stated in the project requirement it should be included here in each object [Refer to the project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=For%20each%20project%20you%20need%20to%20store%20the%20following%20pieces%20of%20data%20in%20a%20JavaScript%20object%2C%20at%20a%20minimum%3A%20name%2C%20description%2C%20featured%20image%2C%20technologies%2C%20link%20to%20live%20version%2C%20link%20to%20source.)